src/Symfony/Component/Mailer/Bridge/Amazon/composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": ">=7.1.3",
-        "symfony/mailer": "^4.4|^5.0"
+        "symfony/mailer": "^4.4.18|^5.1.10|^5.2.1"
```

Comment:
`~5.1.10` instead of `^5.1.10` everywhere

Reply:
Done